src/cleanvision/imagelab.py

```diff
@@ -90,14 +182,55 @@ def _get_issues_to_compute(
     def find_issues(
         self, issue_types: Optional[Dict[str, Any]] = None, n_jobs: Optional[int] = None
     ) -> None:
+        """Finds issues in the dataset.
+        If `issue_types` is not provided, dataset is checked for a default set of issue types.
+        To see default set: :py:meth:`Imagelab.list_default_issue_types`
+
+        Parameters
```

Comment:
`n_jobs` needs documentation. I recommend following documentation from these places:

https://docs.cleanlab.ai/master/cleanlab/filter.html#cleanlab.filter.find_label_issues

> Number of processing threads used by multiprocessing. Default None sets to the number of cores on your CPU (physical cores if you have psutil package installed, otherwise logical cores). Set this to 1 to disable parallel processing (if it's causing issues).